docs/source/references/glossary.md

```diff
@@ -46,3 +46,8 @@ They can  only be build on top of [the Out of the Box Instructions](#out-of-the-
 ## World State View
 
 In-memory representation of the current blockchain state.
+
+## Trigger
+
+Triggers are registered on [peer](#peer) instructions that's execution depends on some conditions,
+for example on the blockchain height, time or [query](#iroha-query) result.
```

Comment:
Triggers are registered on [peer's](#peer) instructions and state that's execution depends on some conditions, for example on the blockchain height, time or [query](#iroha-query) result.
? I really don't understand what this means. Can you clarify what you wanted to say, please?